fcrepo-http-api/src/main/java/org/fcrepo/http/api/FedoraBaseResource.java

```diff
@@ -144,11 +144,11 @@ private String getBaseUrlProperty(final UriInfo uriInfo) {
         return "";
     }
 
-    private HttpSession session() {
-        if (session == null) {
-            throw new SessionMissingException("Invalid session");
+    private Transaction transaction() {
```

Comment:
I understand that you're just replacing the existing method, but I think we can probably drop this. 

For one reason, the transaction is already being injected (from TransactionProvider I assume). But also its only usage is in `new HttpResourceConverter(transaction()`, and it doesn't seem like the new implementation is going to require that. So for the moment I think we can just pass `transaction` directly into `HttpResourceConverter`

Reply:
@bbpennel Can you explain a bit more? This class is using the `transaction()` and the injected `transaction`.

Reply:
@mohideen There were multiple PRs updating this stuff at the same time. Right now on master there is a `getTransaction` method:
https://github.com/fcrepo4/fcrepo4/blob/master/fcrepo-http-api/src/main/java/org/fcrepo/http/api/FedoraBaseResource.java#L155

So what I was suggesting was pulling in the state from master and then either renaming `protected getTransaction()` to `protected transaction()` or just skipping the method all together and having this case and its subclasses directly use the injected `transaction` variable. I'm fine with either. I think the scoping needed to go to `protected` as well, although i don't remember the specifics of why at the moment.

Reply:
Updated to existing transaction associated with the request.

Reply:
Updated to use the `transaction` variable.